Assets/PatchKit Patcher/Scripts/AppUpdater/Commands/CheckVersionIntegrityCommand.cs

```diff
@@ -91,17 +91,20 @@ private FileIntegrity CheckFile(AppContentSummaryFile file)
                 return new FileIntegrity(file.Path, FileIntegrityStatus.MissingMetaData);
             }
 
-            if (_localMetaData.GetEntryVersionId(file.Path) != _versionId)
+            int entryVersionId = _localMetaData.GetEntryVersionId(file.Path);
```

Comment:
Please maybe change `entryVersionId` to `actualEntryVersionId`. It's easier to tell that way what the following if() is doing.